### PR TITLE
POLLS: 2023-10-02_Finalized Drafts for 3 SparkLend Ethereum Polls 

### PR DESCRIPTION
**Pull request by Le-Bateleur-2023**
@VoteWizard ready for review. 

These finalized drafts incorporate Phoenix Labs' final confirmation of Polls configuration.

### CODE REVIEW

File: governance/polls/Increase rETH Supply Cap to 60k - October 2, 2023.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+Express your support or opposition to increasing rETH supply cap from 20k to **60k** on SparkLend Ethereum.
```
Comment (VoteWizard) [inline governance/polls/Increase rETH Supply Cap to 60k - October 2, 2023.md:33]: You could include this link to the Spark Protocol documentation: https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps

File: governance/polls/Onboard USDC to SparkLend Ethereum and Activate USD eMode for USDC and sDAI Markets.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+Express your support or opposition to the following proposed changes:
+
+- Add USDC to SparkLend Ethereum as a borrow-only market, with the below parameters; and
+- Activate USD eMode for USDC and sDAI markets, with the below parameters
```
Comment (VoteWizard) [inline governance/polls/Onboard USDC to SparkLend Ethereum and Activate USD eMode for USDC and sDAI Markets.md:36]: You could include this link to the Spark Protocol documentation:
https://docs.sparkprotocol.io/developers/sparklend/features/efficiency-mode-emode

File: governance/polls/Increase rETH Supply Cap to 60k - October 2, 2023.md (hunk 1 above)
Comment (Le-Bateleur-2023) [reply governance/polls/Increase rETH Supply Cap to 60k - October 2, 2023.md:33]: Done.

**Review by VoteWizard — APPROVED** [review]
I approve this PR

**Pull request merged**